src/gui/src/ServerConfigDialogBase.ui

```diff
@@ -427,50 +427,46 @@ Double click on a screen to edit its settings.</string>
-               <height>21</height>
-              </rect>
-             </property>
-             <property name="text">
-              <string>Max Clipboard sharing size</string>
-             </property>
-            </widget>
-            <widget class="QSpinBox" name="m_pSpinBoxClipboardSizeLimit">
-             <property name="geometry">
-              <rect>
-               <x>250</x>
-               <y>0</y>
-               <width>131</width>
-               <height>20</height>
-              </rect>
-             </property>
-             <property name="toolTip">
-              <string>Clipboard sharing size in bytes</string>
-             </property>
-             <property name="toolTipDuration">
-              <number>-3</number>
-             </property>
-             <property name="alignment">
-              <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
-             </property>
-             <property name="minimum">
-              <number>1000000</number>
-             </property>
-             <property name="maximum">
-              <number>100000000</number>
-             </property>
-             <property name="singleStep">
-              <number>1000</number>
-             </property>
-             <property name="value">
-              <number>3000000</number>
-             </property>
-            </widget>
+            <layout class="QHBoxLayout" name="horizontalLayout">
+             <item>
+              <widget class="QLabel" name="m_pLabelSharingSize">
+               <property name="text">
+                <string>Max Clipboard sharing size</string>
+               </property>
+              </widget>
+             </item>
+             <item>
+              <widget class="QSpinBox" name="m_pSpinBoxClipboardSizeLimit">
+               <property name="sizePolicy">
+                <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
+                 <horstretch>0</horstretch>
+                 <verstretch>0</verstretch>
+                </sizepolicy>
+               </property>
+               <property name="toolTip">
+                <string>Clipboard sharing size in bytes</string>
+               </property>
+               <property name="toolTipDuration">
+                <number>-3</number>
+               </property>
+               <property name="alignment">
+                <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
+               </property>
+               <property name="minimum">
+                <number>1000000</number>
+               </property>
+               <property name="maximum">
+                <number>100000000</number>
+               </property>
+               <property name="singleStep">
+                <number>1000</number>
+               </property>
+               <property name="value">
+                <number>3000000</number>
+               </property>
+              </widget>
+             </item>
+            </layout>
```

Comment:
This looks fine, but does it work with Qt 5?

Reply:
It should work just fine with both Qt5 and Qt6

Reply:
Awesome. LGTM.